src/Features/ERDDAP/Platform/Observations/WindCondition/index.tsx

```diff
@@ -1,90 +1,90 @@
+import { PlatformFeature, PlatformTimeSeries } from "../../../types"
+import { pickWindDatasets, pickWindTimeSeries } from "../../../utils/wind"
+import { UseDatasets } from "Features/ERDDAP/hooks"
 
 interface Props {
-  platform: PlatformFeatureWithDatasets
-  unit_system: UnitSystem
+  platform: PlatformFeature
 }
 
-export const windStandards = new Set(["wind_from_direction", "wind_speed", "wind_speed_of_gust"])
-
-export const ErddapWindObservedConditionBase: React.SFC<Props & SizeMeProps> = ({ platform, size, unit_system }) => {
-  // adjust number of barbs based on width
-  let barbsPerDay = 5
-  if (size && size.width && size.width < 800) {
-    barbsPerDay = 2
-  }
-
-  const windDatasets = platform.properties.readings.filter((reading) =>
-    windStandards.has(reading.data_type.standard_name)
-  )
+interface DisplayProps extends Props {
+  unit_system: UnitSystem
+  datasets: DataTimeSeries[]
+  timeSeries: PlatformTimeSeries[]
+}
 
-  const windTimeSeries: DataTimeSeries[] = windDatasets.map((reading) => ({
-    name: reading.data_type.long_name,
-    timeSeries: reading.readings,
-    unit: reading.data_type.units,
-  }))
+/**
+ * Wind Observed conditions component
+ */
+export const ErddapWindObservedCondition: React.FunctionComponent<Props> = ({ platform }) => {
+  const unit_system = useUnitSystem()
 
-  Sentry.addBreadcrumb({
-    category: "ERDDAP Wind",
-    data: {
-      platformId: platform.id as string,
-      windDatasets,
-    },
-  })
+  const { timeSeries } = pickWindTimeSeries(platform)
 
-  if (windDatasets.length === 0) {
-    Sentry.captureMessage(("Unable to display wind data for " + platform.id) as string)
-    return noWind
+  if (timeSeries.length < 0) {
+    return <WindError message="No wind data" />
   }
 
   return (
-    <React.Fragment>
-      <ErddapDatasetStatus datasets={windDatasets} />
-      <ErddapDatasetLoader platformId={platform.id as string} datasets={windDatasets}>
-        <React.Fragment>
-          {windTimeSeries[0].timeSeries !== undefined && windTimeSeries[0].timeSeries.length > 0 ? (
-            <WindChart barbsPerDay={barbsPerDay} data={windTimeSeries} unit_system={unit_system} />
-          ) : null}
-        </React.Fragment>
-      </ErddapDatasetLoader>
-    </React.Fragment>
+    <UseDatasets
+      timeSeries={timeSeries}
+      error={<WindError message="Error loading wind data" />}
+      loading={
+        <Row>
+          <Col>
+            <h5>Loading wind data in progress</h5>
+          </Col>
+        </Row>
+      }
+    >
+      {({ datasets }) => <ErddapWindObservedConditionDisplay {...{ platform, unit_system, timeSeries, datasets }} />}
+    </UseDatasets>
   )
 }
 
-const noWind = (
-  <Row>
-    <Col>
-      <Alert color="warning">Unable to display wind info</Alert>
-    </Col>
-  </Row>
-)
+/**
+ * Display wind datasets
+ */
+export const ErddapWindObservedConditionDisplay: React.FunctionComponent<DisplayProps> = ({
+  platform,
+  unit_system,
```

Comment:
Codacy found an issue: [Identifier 'unit_system' is not in camel case.](https://app.codacy.com/manual/gmri/Neracoos-1-Buoy-App/pullRequest?prid=6166252)